website/docs/migrating-from-docker/managing-docker-compatibility.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 5
+title: Managing Docker compatibility
+description: Covers configurations available to manage docker compatibility
+keywords: [podman desktop, podman, docker, compatibility]
+tags: [managing-docker-compatibility]
+---
+
+# Managing Docker compatibility
+
+With Podman Desktop, you can configure a Docker-compatible environment to run your Docker applications on a Podman engine, rather than on a Docker engine.
+
```

Comment:
it's not clear to me if it's to have a docker-compatible environment and then using podman CLI, etc

or to use docker tooling against the podman engine.

It's both but maybe it needs to be more highlighted